### PR TITLE
MCR-1826 migrated to match changes in MyCoRe

### PR DESCRIPTION
**Pull request by paschty**

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 21c05551251105e9f336d8984b88ac9758f8040c! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1525-pull_request-269

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with bac6aa13111a5eb1250afd73dc3c1de3050b3767! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1527-pull_request-269

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with a809fc4eeb58f9ed14711f4c9621a7bea56770aa! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1539-pull_request-269

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with a809fc4eeb58f9ed14711f4c9621a7bea56770aa! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1539-pull_request-269

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with b9339193feda1c57a52fe9bbe8a8aae2c9e74aad! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1541-pull_request-269

**Comment by kkrebs** [comment]
please fix build error:
[ERROR] Could not find goal 'installConfigSet' in plugin org.mycore.plugins:solr-runner-maven-plugin:1.1-SNAPSHOT among available goals copyHome, installCoreTemplate, installSolrPlugins, start, stop -> [Help 1]

File: mir-webapp/pom.xml [hunk 1]
```diff
@@ -89,12 +90,14 @@
...
         <groupId>org.mycore.plugins</groupId>
         <artifactId>solr-runner-maven-plugin</artifactId>
-        <version>1.0</version>
+        <version>1.1-SNAPSHOT</version>
         <configuration>
           <solrHomeTemplate>src/main/solr</solrHomeTemplate>
           <solrHome>${mir.solr.home}</solrHome>
           <solrMirrorURL>${solr-runner.solrMirrorURL}</solrMirrorURL>
           <solrVersionString>${solr-runner.solrVersionString}</solrVersionString>
+          <gitRepository>https://github.com/MyCoRe-Org/mycore_solr_configset_main.git</gitRepository>
```
Comment (yagee-de) [inline mir-webapp/pom.xml]: we should stick to the mir-solr submodule. please make a branch there with the configsets (further submodules?) and use this branch here. There is a need to support at least 2 different config sets: "mir" and "classification". Sticking to the submodule does not require a new release of the solr-runner-maven-plugin.

File: pom.xml [hunk 2]
```diff
@@ -50,7 +50,7 @@
     <selenium.version>3.8.1</selenium.version>
     <site.suffix />
     <solr-runner.solrMirrorURL>http://mirror.23media.de/apache/lucene/solr/</solr-runner.solrMirrorURL>
-    <solr-runner.solrVersionString>7.2.1</solr-runner.solrVersionString>
+    <solr-runner.solrVersionString>7.0.1</solr-runner.solrVersionString>
```
Comment (yagee-de) [inline pom.xml]: please check if 7.3.1 works

File: mir-it/pom.xml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (yagee-de) [inline mir-it/pom.xml]: please use "-Pauto-update" to format pom.xml

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 848030ac3cf186824704003bd2574b6883fe1771! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1553-pull_request-269

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 6821dfac87e55dce44cab965815f16e72864c208! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1555-pull_request-269

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 6821dfac87e55dce44cab965815f16e72864c208! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1555-pull_request-269

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 552d6fe44392c86ed0d0d2df123518e593ba5173! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1569-pull_request-269

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 9351ce6f35f39bd320e8d1ae91ba1a83da1e6ef6! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1574-pull_request-269

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with afb87303be0da46760f0963e3a3169eef0719aed! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1576-pull_request-269

**Pull request merged**